sysdump/sysdump.go

```diff
@@ -679,6 +681,38 @@ func (c *Collector) Run() error {
 				return nil
 			},
 		},
+		{
+			Description: "Collecting the Cilium Envoy configuration",
+			Quick:       true,
+			Task: func(ctx context.Context) error {
+				v, err := c.Client.GetConfigMap(ctx, c.Options.CiliumNamespace, ciliumEnvoyConfigMapName, metav1.GetOptions{})
+				if err != nil {
+					return fmt.Errorf("failed to collect the Cilium Envoy configuration: %w", err)
+				}
+				if err := c.WriteYAML(ciliumEnvoyConfigMapFileName, v); err != nil {
+					return fmt.Errorf("failed to collect the Cilium Envoy configuration: %w", err)
+				}
+				return nil
+			},
+		},
+		{
+			Description: "Collecting the Cilium Envoy daemonset",
+			Quick:       true,
+			Task: func(ctx context.Context) error {
+				v, err := c.Client.GetDaemonSet(ctx, c.Options.CiliumNamespace, ciliumEnvoyDaemonSetName, metav1.GetOptions{})
+				if err != nil {
+					if errors.IsNotFound(err) {
+						c.logWarn("Daemonset %q not found in namespace %q - this is expected if Envoy DaemonSet is not enabled", ciliumEnvoyDaemonSetName, c.Options.CiliumNamespace)
```

Comment:
Do we usually warn in this context? This seems like an info would be better (at least for now, since it is optional)

Reply:
@squeed good point - i agree with you. but it looks like this is kind of the "default behaviour" for optional components - see [clustermesh-apiserver](https://github.com/cilium/cilium-cli/blob/main/sysdump/sysdump.go#L771), [hubble](https://github.com/cilium/cilium-cli/blob/main/sysdump/sysdump.go#L721), [hubble-ui](https://github.com/cilium/cilium-cli/blob/main/sysdump/sysdump.go#L739)